webview-ui/src/context/ExtensionStateContext.tsx

```diff
@@ -145,6 +152,10 @@ export const ExtensionStateContextProvider: React.FC<{
 		vscode.postMessage({ type: "webviewDidLaunch" })
 	}, [])
 
+	const requestTotalTasksSize = useCallback(() => {
+		vscode.postMessage({ type: "requestTotalTasksSize" })
```

Comment:
instead of creating `requestTotalTasksSize` lets just postMessage in the HistoryView useEffect